hack/local-up-cluster.sh

```diff
@@ -202,6 +202,18 @@ EXPERIMENTAL_CRI=${EXPERIMENTAL_CRI:-"false"}
 # Ensure CERT_DIR is created for auto-generated crt/key and kubeconfig
 mkdir -p "${CERT_DIR}" &>/dev/null || sudo mkdir -p "${CERT_DIR}"
 CONTROLPLANE_SUDO=$(test -w "${CERT_DIR}" || echo "sudo -E")
+# If on an selinux system, need to set the correct context so cfssl container can access it
+if command -v getenforce &>/dev/null; then
+    if [[ "$(getenforce)" != "Disabled" ]]; then
+        # Try newer container_file_t type first
+        if ! sudo chcon -R -t container_file_t "${CERT_DIR}"; then
```

Comment:
@stevekuznetsov I see what you mean.  I think the window for that is small just because 1) we are running as root so shouldn't hit permissions errors 2) we know the directory exists and 3) we know we are on a system that likely has an selinux policy installed with the presence of getenforce.

This script is run in a variety of environments/distros so using the distro package manager in it is probably a not allowed.

IMHO I think it is good enough the way it is.  Thanks for the great bash feedback! Made the code much more pleasant to look at :)

Could I get an lgtm if you agree? All tests are green. /cc @sttts